terraform/cloudfront.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_cloudfront_distribution" "data_api" {
+  enabled             = true
+  is_ipv6_enabled     = true
+  price_class         = "PriceClass_All"
+  aliases             = [replace(var.service_url, "https://", "")]
+
+  origin {
+    domain_name = module.fargate_autoscaling.lb_dns_name
+    custom_origin_config {
+      http_port = 80
+      https_port = 443
+      origin_keepalive_timeout = 5
+      origin_protocol_policy = "http-only"
```

Comment:
we have to use `http-only` for S3 origin. Here we are using a load balancer which supports https protocol and we should use that.

More info:
https://aws.amazon.com/premiumsupport/knowledge-center/cloudfront-https-connection-fails/

PS: I realize the tile server has the same config. It must have slipped through but we should also change that.

Reply:
PPS: I realize that currently our load balancer uses the acm certificate and this will create issues when we switch domain names. So it is actually best to use `http-only` first. Only after domains are switched we detach the gfw certificate from the load balancer and use the default aws one. at that point we might want to switch to https end-to-end